### DISCUSSION TITLE
Fixing a corrupted outline deletes rows — no backup.

### DISCUSSION THREAD

**Discussion by bobembry** [vincode-io/Zavala#257] [Support]
Yesterday I had an outline that displayed correctly on my MacBook Air and iPhone. 

The same outline was listed on my iPad but the outline rows were completely missing — only the title and tags were visible.

I created a new row on my MacBook Air to see if that would force a refresh on my iPad. 

After a few seconds I got a notification on my iPad that the outline had been corrupted  and given the option to fix it. 

I selected the fix it option and the rows on my MacBook Air and iPhone were deleted. 

The only remaining row was the one I created to refresh the iPad display. After this all three devices were in sync

Any suggestions?

Thanks

Bob

**Comment by vincode-io** [comment]
First of all, let me apologize for this happening. Data loss should never happen in an app.

Please email me directly using the Provide Feedback option in Zavala and I'll do my best to recover any data that we can.

**Reply by bobembry** [reply]
Email sent through Feedback
